app/test_main.py

```diff
@@ -1,3 +1,61 @@
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
```

Comment:
According to the checklist: 'if output should change with some integer value you should check that it is not changed with previous value'. It seems like you are not following this point. For example, the test case where cat_age and dog_age are 14, and the result is [0, 0] is not needed, because it tests the same case as the test case where cat_age and dog_age are 0.